### DISCUSSION TITLE
Axis 2.x

### DISCUSSION THREAD

**Discussion by ISIthayden** [eclipse-birt/birt#1930] [General]
Are there any plans or intentions to migrate BIRT to use Axis 2.x in the future? While I have seen that the CVEs with Axis 1.4 have been dismissed as either not applicable or fixed with 1.4.1, I have a usecase that will not allow me to move forward with any CVEs being caught.

**Comment by merks** [comment]
As far as I know, there are no such plans. BIRT is using a recently-built version of axis with the CVEs addressed. But if someone says that’s not the case, that’s their choice to say.

**Reply by ISIthayden** [reply]
thank you for the response
